### PR TITLE
Adds Pilates Anytime

### PR DESCRIPTION
**Pull request by alex-parsons**
Adds the following events or companies:
 - Pilates Anytime 

### Checklist

 - [X] I have updated the event or company counts
 - [X] I have put the most recent relevant date in the "Last Update" column
 - [] I have linked to the article about the change, not the company's website

### CODE REVIEW

**Comment by drewbrew** [comment]
Do you have a public-facing post by chance for this chance?

**Comment by alex-parsons** [comment]
We do not at this time; I will create another PR if this happens.

**Comment by drewbrew** [comment]
That works. Thanks!

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 This is the running list of what in tech has been affected by COVID-19. Pull requests gratefully accepted, especially around design or data formatting or correctness.
 
 <a name="companies"></a>
-## Companies - 161
+## Companies - 162
```
Comment (drewbrew) [inline README.md]:
```suggestion
## Companies - 163
```

**Pull request merged**